pkg/model/loader.go

```diff
@@ -40,8 +41,9 @@ func (d defaultLoader) Load(path string) (*Model, error) {
 		return nil, err
 	}
 
+	// Load the model from <path>/model.
 	buf := &bytes.Buffer{}
-	if err := Tar(path, buf); err != nil {
+	if err := Tar(filepath.Join(path, consts.ORMBModelDirectory), buf); err != nil {
```

Comment:
Should we check the directory format? To ensure that the format in the ormbfile is consistent with what is being passed

Reply:
I think it is a huge feature. We can open an issue and implement it in the future. WDYT

Reply:
LGTM

issue created https://github.com/caicloud/ormb/issues/11